roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -49,52 +39,18 @@ if [[ $2 =~ ^(up|dhcp4-change|dhcp6-change)$ ]]; then
   if [[ ${DEVICE_IFACE} == ${def_route_int} ]]; then
     if [ ! -f /etc/dnsmasq.d/origin-dns.conf ]; then
       cat << EOF > /etc/dnsmasq.d/origin-dns.conf
-no-resolv
 domain-needed
-server=/cluster.local/172.30.0.1
-server=/30.172.in-addr.arpa/172.30.0.1
 enable-dbus
+bind-dynamic
+except-interface=lo
```

Comment:
I was thinking we might want to make except-interface configurable.  Today it listens only on one interface (the public interface), however with bind-dynamic we will listen on every interface.  Some users might have machines with multiple lans, and it might not be desireable for them to bind to those interfaces.

I would make this a template, accept a user-provided list, and join into a csv or space separated, not sure what dns mask takes here.